src/main/kotlin/no/nav/syfo/melding/database/domain/PMelding.kt

```diff
@@ -26,7 +26,10 @@ data class PMelding(
     val journalpostId: String?,
     val ubesvartPublishedAt: OffsetDateTime?,
     val veilederIdent: String?,
-)
+) {
+    @JvmInline
+    value class Id(val id: Int)
+}
```

Comment:
Vi prøvde ut en liten endring her: En måte å være enda strengere på at denne verdien skal være en `Id`, ikke bare en `Int`. Det gjør at vi også rundt omkring kan skrive `PMelding.Id` for å vise hva vi referer til i feks `PMeldingStatus` som har en `PMelding.Id` som foreign key.